src/test/java/org/dependencytrack/tasks/metrics/PortfolioMetricsUpdateTaskTest.java

```diff
@@ -62,6 +69,12 @@ public static void tearDownClass() {
 
     @Test
     public void testUpdateMetricsEmpty() {
+        qm.createConfigProperty(CUSTOM_RISK_SCORE_CRITICAL.getGroupName(), CUSTOM_RISK_SCORE_CRITICAL.getPropertyName(), "10", IConfigProperty.PropertyType.INTEGER, null);
+        qm.createConfigProperty(CUSTOM_RISK_SCORE_HIGH.getGroupName(), CUSTOM_RISK_SCORE_HIGH.getPropertyName(), "5", IConfigProperty.PropertyType.INTEGER, null);
+        qm.createConfigProperty(CUSTOM_RISK_SCORE_MEDIUM.getGroupName(), CUSTOM_RISK_SCORE_MEDIUM.getPropertyName(), "3", IConfigProperty.PropertyType.INTEGER, null);
+        qm.createConfigProperty(CUSTOM_RISK_SCORE_LOW.getGroupName(), CUSTOM_RISK_SCORE_LOW.getPropertyName(), "1", IConfigProperty.PropertyType.INTEGER, null);
+        qm.createConfigProperty(CUSTOM_RISK_SCORE_UNASSIGNED.getGroupName(), CUSTOM_RISK_SCORE_UNASSIGNED.getPropertyName(), "5", IConfigProperty.PropertyType.INTEGER, null);
```

Comment:
Consider replacing literals (e.g. `"10"`, `IConfigProperty.PropertyType.INTEGER`) with the properties' getters instead. That ensures that changes made to `ConfigPropertyConstants` also apply to tests.

So, instead of:

```java
qm.createConfigProperty(CUSTOM_RISK_SCORE_CRITICAL.getGroupName(), CUSTOM_RISK_SCORE_CRITICAL.getPropertyName(), "10", IConfigProperty.PropertyType.INTEGER, null);
```

Consider doing:

```
qm.createConfigProperty(
    CUSTOM_RISK_SCORE_CRITICAL.getGroupName(), 
    CUSTOM_RISK_SCORE_CRITICAL.getPropertyName(), 
    CUSTOM_RISK_SCORE_CRITICAL.getDefaultPropertyValue(), 
    CUSTOM_RISK_SCORE_CRITICAL.getPropertyType(), 
    CUSTOM_RISK_SCORE_CRITICAL.getDescription()
);
```

If this takes too much visual space, consider moving it to a method in `AbstractMetricsUpdateTaskTest`, so all `*MetricsUpdateTaskTest` classes can make use of it.

Reply:
updated to use the property getters and used a helper function to create the properties